uaa-server/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM adoptopenjdk/openjdk11 as source
+RUN apt-get update && apt-get install -y git
+RUN git clone -b v75.0.0 https://github.com/cloudfoundry/uaa.git
+COPY uaa.yml /uaa/uaa/src/main/resources/uaa.yml
+WORKDIR /uaa
+RUN /uaa/gradlew
```

Comment:
By not specifying a Gradle task, the only work being done here is the resolution of project dependencies.  Project compilation won't occur until `run` task below, thus compilation will occur every time container is started.

I'm not 100% of the best task here, but it's likely `assemble` or `war`.  Compiling here should shorten startup times.